src/components/data-entry/DataSetStateButton.tsx

```diff
@@ -52,9 +55,14 @@ const DataSetStateButton: React.FunctionComponent<DataSetStateButtonProps> = pro
     }, [projectDataSet, notifyOnChange, showErrorAndSetInactive, validation]);
 
     const notifyUsers = React.useCallback(async () => {
-        const notificator = new ProjectNotification(api, project, currentUser, isTest);
-        await notificator.notifyOnDataReady(period, dataSet.id);
-    }, [api, project, currentUser, isTest, period, dataSet.id]);
+        try {
+            const notificator = new ProjectNotification(api, project, currentUser, isTest);
+            await notificator.notifyForDataReview(period, dataSet.id, dataSetType);
+            snackbar.success(i18n.t("The email has been successfully sent to data reviewers."));
+        } catch (err) {
+            snackbar.error(i18n.t("There was an error notifying the data reviewers."));
```

Comment:
I've added here `err.message` to get a specific message instead of a generic one.